src/library-authoring/author-library/LibraryAuthoringPage.jsx

```diff
@@ -335,146 +335,154 @@ export const LibraryAuthoringPageBase = ({
-                  <Col xs={12} md={9} className="pb-2">
-                    <SearchField
-                      label={intl.formatMessage(messages['library.detail.search'])}
-                      value={query}
-                      onSubmit={(value) => changeQuery(value)}
-                      onChange={(value) => changeQuery(value)}
-                    />
-                  </Col>
-                  <Col xs={12} md={3} className="pb-2">
-                    <Input
-                      type="select"
-                      data-testid="filter-dropdown"
-                      value={type}
-                      options={typeOptions}
-                      onChange={(event) => changeType(event.target.value)}
-                    />
-                  </Col>
-                </>
-              )}
-              <Col xs={12} className="text-center d-md-none py-3">
-                <ButtonToggles
-                  setShowPreviews={setShowPreviews}
+            <ActionRow className='component-search-and-filter'>
+            {(library.type === LIBRARY_TYPES.COMPLEX) && (
+              <>
+                <SearchField
+                  className='content-library-search-box'
+                  value={query}
+                  placeholder={intl.formatMessage(messages['library.detail.search'])}
+                  onSubmit={(value) => changeQuery(value)}
+                  onChange={(value) => changeQuery(value)}
+                />
+                <ActionRow.Spacer />
+                <Form.Control
+                  className='component-type-filter-dropdown'
+                  as='select'
+                  data-testid='filter-dropdown'
+                  value={type}
+                  onChange={(event) => changeType(event.target.value)}
+                >
+                  {typeOptions.map(typeOption => (
+                    <option value={typeOption.value}>{typeOption.label}</option>
+                  ))}
+                </Form.Control>
+              </>
+            )}
+            </ActionRow>
+            {/* todo: figure out how we want to handle these at low screen widths.
+                      mobile is currently unsupported: so it doesn't make sense
+                      to have partially implemented responsive logic */}
+            {/* <Col xs={12} className="text-center d-md-none py-3">
+              <ButtonToggles
+                setShowPreviews={setShowPreviews}
+                showPreviews={showPreviews}
+                library={library}
+                sending={sending}
+                quickAddBehavior={quickAddBehavior}
+                className="d-md-none py-3"
+              />
+            </Col> */}
+            <LoadGuard
+              loadingMessage={intl.formatMessage(messages['library.detail.loading.message'])}
+              condition={blocks.status !== LOADING_STATUS.LOADING}
+            >
+              {() => blocks.value.data.map((block) => (
+                <BlockPreviewContainer
```

Comment:
We're missing a `key={}` here: React is throwing a console error because of it.

Reply:
turns out it was somewhere else (thanks for finding it!) https://github.com/brian-smith-tcril/frontend-app-library-authoring/commit/b56b7d2adf60db4c4e91da77eb8923030f907b33